### PR TITLE
Release 1.7.0

### PR DESCRIPTION
**Pull request by Kircheneer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## v1.7.0 - 2022-10-03
```
Comment (itdependsnetworks) [inline CHANGELOG.md:3]: 11-03?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+
+### Changed
+
+- #176 - Remove pytest-redislite in favor of pytest-redis.
+- #174 - Update Dockerfile to install build-essential
+
+### Added
+
+- #174 - Add methods to load data from dictionary and enable tree traversal
```
Comment (itdependsnetworks) [inline CHANGELOG.md:12]: Get or none as well

File: CHANGELOG.md (hunk 1 above)
Comment (Kircheneer) [reply CHANGELOG.md]: Thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (Kircheneer) [reply CHANGELOG.md:12]: Thanks!

**Review by itdependsnetworks — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+- #168 - Add 'skip' counter to diff.summary()
+- #169/#170 - Add documentation about model processing order
+- #121/#140 - Add and configure renovate
+- #140 - Add renovate configuration validation to the CI
```
Comment (chadell) [inline CHANGELOG.md:17]: why #140 is duplicated?

Comment (chadell) [reply CHANGELOG.md:17]: maybe because it tackles different topics? (similar to #174)

Comment (Kircheneer) [reply CHANGELOG.md:17]: Exactly, I wanted to highlight all the individual things that were done there.

**Review by chadell — APPROVED** [review]
LGTM

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**